src/Microsoft.ML.Api/CodeGenerationUtils.cs

```diff
@@ -1,142 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using System.Text;
-using System.Text.RegularExpressions;
-using System.CodeDom;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.CSharp;
-using System.IO;
-
-namespace Microsoft.ML.Runtime.Api
-{
-    /// <summary>
-    /// Utility methods for code generation.
-    /// </summary>
-    internal static class CodeGenerationUtils
```

Comment:
>CodeGenerationUtils [](start = 26, length = 19)

you need to do something with

 [Fact]
        public void CommandCodeGen()
        {
            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                return;

Reply:
Delete it.

Reply:
yes, saw that failing. thanks. 

---
In reply to: [240723443](https://github.com/dotnet/machinelearning/pull/1848#discussion_r240723443) [](ancestors = 240723443)